src/core/schema/config.model.schema

```diff
@@ -291,6 +291,19 @@
     },
     "themeSettings": {
       "type": "object"
+    },
+    "_inview": {
+      "type": "object",
+      "title": "jquery.onscreen (inview)",
```

Comment:
```suggestion
      "title": "",
```
The extra information is probably not too useful for the average user – the key is already transformed into the main title. With the current schema we'd get a double title:
![double](https://user-images.githubusercontent.com/922987/52790800-67064000-305f-11e9-9b48-609dd85584b9.png)